setup.py

```diff
@@ -49,6 +49,9 @@ def read(fname, URL, URLImage):
     long_description=read('README.rst', about['__github_url__'], about[
         '__github_assets_absoluteURL__']),
     url=about['__url__'],
+    project_urls={
+        'Source': 'https://github.com/MechanicalSoup/MechanicalSoup',
```

Comment:
We already have the string in `__version__.py` to make it available to the source code itself. To avoid duplicating, it should be:

```suggestion
        'Source': about['__github_url__'],
```